packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-VisualBasic/Class1.vb

```diff
@@ -0,0 +1,3 @@
+Public Class Class1

```

Comment:
This is missing `Namespace`

Reply:
VB specified a root namespace and all files in the project are automatically a part of that namespace.  This language behavior is different from how C# behaved.  VB programmers are not used to needing to place code inside namespaces

Reply:
Ah ok - thanks for the clarification.